### ISSUE TITLE
simple test case failed

### ISSUE THREAD

**Issue by qiaojingx** [01org/zephyr.js#852]
### Description

simple test case failed

### Test Code

[test-buffer.js](https://github.com/01org/zephyr.js/blob/master/tests/test-buffer.js); 
 
### Steps to Reproduction

1. create a simple test case as below:
![selection_004](https://cloud.githubusercontent.com/assets/19813546/24140441/06c7efe0-0e5b-11e7-9237-428963a4cb85.png)
2. Build this simple test case and flash to A101.
3. Thrown error.
![selection_005](https://cloud.githubusercontent.com/assets/19813546/24140473/28821872-0e5b-11e7-848e-d97b6ddfd758.png)
 
### Actual Result

1. Run test code [test-buffer.js](https://github.com/01org/zephyr.js/blob/master/tests/test-buffer.js) on Arduino 101, FRDM-K64F:
![selection_001](https://cloud.githubusercontent.com/assets/19813546/24140406/e12d651c-0e5a-11e7-9d8e-99d83426516f.png)
 
### Expected Result

No error thrown
 
### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | 02d9471 | Arduino 101 | Mar 21, 2017 | Fail
master | 02d9471 | FRDM-K64F | Mar 21, 2017 | Fail
master | 02d9471 | Linux | Mar 21, 2017 | Pass

**Comment by brianjjones** [comment]
@grgustaf I don't think we support this do we?  I don't recall us having any of the Math functions.

**Comment by grgustaf** [comment]
We have always left out the Math library from JS to save space. However, we do have enough room these days that we could enable it, although it is large and would limit what other modules could be used. So this is basically a feature request, not a bug.

**Comment by jprestwo** [comment]
We do have an easy-ish way to enable it. You can edit `jerry_feature.profile` and comment out the `CONFIG_DISABLE_MATH_BUILTIN` line. Then if you rebuild it will include the Math object.

**Comment by kenchris** [comment]
In ES2017, ** is a default operator for Math.pow, so at some point is should be supported even if the Math functions aren't

**Comment by rwaldron** [comment]
@kenchris 7th Edition: ES2016\* ;) I'm curious though—what is JerryScript's roadmap? I can't find anything anywhere. I'm going to file an issue over there to see what we can learn.

**Comment by rwaldron** [comment]
@kenchris https://github.com/jerryscript-project/jerryscript/issues/1735

**Comment by kenchris** [comment]
Right, I meant ES2016, the release with only two new features ;)

**Comment by rwaldron** [comment]
Lol, that was the best release ever :)
